TheForceEngine/TFE_Settings/settings.cpp

```diff
@@ -1483,4 +1503,37 @@ namespace TFE_Settings
 		}
 		free(data);
 	}
+
+	void loadCustomLogics()
+	{
+		vector<string> jsons;
+		TFE_Paths::getAllFilesFromSearchPaths("logics", "json", jsons);
+
+		for (u32 i = 0; i < jsons.size(); i++)
+		{
+			TFE_System::logWrite(LOG_MSG, "LOGICS", "Parsing logic JSON");
```

Comment:
Do we need to write log spam for each logic? It might be better to write at the top of the loop. You can write the number parsed at the end if desired.

Reply:
Once you move this out of the loop, I will squash and push the changes. :)

Thanks for adding this feature!